test/ContainerTrait.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive;
+
+use Interop\Container\ContainerInterface;
+use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
+
+/**
+ * Helper methods for mock Interop\Container\ContainerInterface.
+ */
+trait ContainerTrait
+{
+    /**
+     * Returns a prophecy for ContainerInterface.
+     *
+     * By default returns false for unknown `has('service')` method.
+     *
+     * @return ObjectProphecy
+     */
+    protected function mockContainerInterface()
+    {
+        $container = $this->prophesize(ContainerInterface::class);
+        $container->has(Argument::type('string'))->willReturn(false);
+
+        return $container;
+    }
+
+    /**
+     * Add a service to the container.
+     *
+     * Adjust `has('service')` and `get('service')` returns.
+     *
+     * @param ObjectProphecy $container
+     * @param string $serviceName
+     * @param mixed $service
+     *
+     * @return void
+     */
+    protected function setServiceToContainer(ObjectProphecy $container, $serviceName, $service)
```

Comment:
This method name does not make sense to this particular native English speaker. Can you please switch it to one of these possibilities?

- injectServiceInContainer
- setContainerService

Obviously, this will require changes in the test cases where the method is called.